### DISCUSSION TITLE
COST UE cannot connect to Internet

### DISCUSSION THREAD

**Discussion by chonghua1** [srsran/srsRAN_Project#576] [General Help]
I have set up a 5G SA system with USRP B210 and Goole pixel7A, the whole system worked well, but today when I connect the UE to gNB, the phone can't reach the internet. I have added the NAT rule:
```
### Enable IPv4/IPv6 Forwarding
$ sudo sysctl -w net.ipv4.ip_forward=1
$ sudo sysctl -w net.ipv6.conf.all.forwarding=1

### Add NAT Rule
$ sudo iptables -t nat -A POSTROUTING -s 10.45.0.0/16 ! -o ogstun -j MASQUERADE
$ sudo ip6tables -t nat -A POSTROUTING -s 2001:db8:cafe::/48 ! -o ogstun -j MASQUERADE
```
the amf log:
![image](https://github.com/srsran/srsRAN_Project/assets/42130689/d1eff04b-f487-4548-a51f-541d4a602874)
the upf log:
![image](https://github.com/srsran/srsRAN_Project/assets/42130689/8dab4a14-91c1-4241-bebf-7f90f96b3dc6)
![image](https://github.com/srsran/srsRAN_Project/assets/42130689/19fd99fa-0253-4fc8-b187-58e7ad521c3b)

**Comment by pgawlowicz** [comment]
could you share your configs? could you reduce tx gain a bit?

**Reply by chonghua1** [reply]
Thanks for the reply! I have solved the problem with disable the firewall :D
